// From the google example https://developers.google.com/protocol-buffers/docs/pythontutorial.
syntax = "proto3";

package modulus.modulusobject;

message MaglevPerson {
    string full_name = 1;
    int32 id = 2;
    string email = 3;

    enum MaglevPhoneType {
        MOBILE = 0;
        HOME = 1;
        WORK = 2;
    }

    message MaglevPhoneNumber {
        string number = 1;
        MaglevPhoneType type = 2;
    }

    repeated MaglevPhoneNumber phones = 4;
}

message MaglevAddressBook { repeated MaglevPerson people = 1; }